lib/smack/SmackRep.cpp

```diff
@@ -154,8 +210,8 @@ void SmackRep::collectRegions(llvm::Module &M) {
 
 const Stmt* SmackRep::alloca(llvm::AllocaInst& i) {  
   const Expr* size = 
-    Expr::fn("$mul",lit(storageSize(i.getAllocatedType())),lit(i.getArraySize()));
-                       
+    Expr::fn(opName("$mul", {TRUNC(getIntSize(i.getOperand(0)))}), lit(storageSize(i.getAllocatedType())),lit(i.getArraySize()));
```

Comment:
I guess this `TRUNC` is forcing `size_t` to be interpreted as `i32`. It's not clear this is a good idea though.